### PR TITLE
Update metatags of Kafka and FTP BBEs

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$subject

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/ftp-service-send-file/ftp_service_send_file.metatags [hunk 1]
```diff
@@ -1,2 +1,2 @@
-description: This BBE is about receiving file/directory changes that occur in a remote FTP server and using the `ftp:Caller` to interact with the FTP server.
+description: This example is about receiving file/directory changes that occur in a remote FTP server and using the `ftp:Caller` to interact with the FTP server.
```
Comment (praneesha) [inline examples/ftp-service-send-file/ftp_service_send_file.metatags:1]:
```suggestion
description: This example demonstrates receiving file/directory changes that occur in a remote FTP server and using an ftp:Caller to interact with the FTP server.
```
It gives a syntax error when we use backquotes in metadata.

File: examples/kafka-consumer-consumer-record-data-binding/kafka_consumer_consumer_record_data_binding.metatags [hunk 2]
```diff
@@ -1,2 +1,2 @@
-description: BBE on how to use a `kafka:Consumer` as a simple record consumer.
+description: This example shows how to use a `kafka:Consumer` as a simple record consumer.
```
Comment (praneesha) [inline examples/kafka-consumer-consumer-record-data-binding/kafka_consumer_consumer_record_data_binding.metatags:1]:
```suggestion
description: This example demonstrates how to use a kafka:Consumer as a simple record consumer.
```
Changed for consistency.

File: examples/kafka-consumer-payload-data-binding/kafka_consumer_payload_data_binding.metatags [hunk 3]
```diff
@@ -1,2 +1,2 @@
-description: BBE on how to use a `kafka:Consumer` as a payload consumer.
+description: This examples shows how to use a `kafka:Consumer` as a payload consumer.
```
Comment (praneesha) [inline examples/kafka-consumer-payload-data-binding/kafka_consumer_payload_data_binding.metatags:1]:
```suggestion
description: This example demonstrates how to use a kafka:Consumer as a payload consumer.
```

File: examples/kafka-producer-produce-message/kafka_producer_produce_message.metatags [hunk 4]
```diff
@@ -1,2 +1,2 @@
-description: BBE on sending messages to a Kafka topic using a `kafka:Producer` client.
+description: This example is on sending messages to a Kafka topic using a `kafka:Producer` client.
```
Comment (praneesha) [inline examples/kafka-producer-produce-message/kafka_producer_produce_message.metatags:1]:
```suggestion
description: This example demonstrates sending messages to a Kafka topic using a kafka:Producer client.
```

File: examples/kafka-service-consume-message/kafka_service_consume_message.metatags [hunk 5]
```diff
@@ -1,2 +1,2 @@
-description: BBE on creating a Kafka service to receive messages in an asynchronous manner using Ballerina.
+description: This examples is on creating a Kafka service to receive messages in an asynchronous manner using Ballerina.
```
Comment (praneesha) [inline examples/kafka-service-consume-message/kafka_service_consume_message.metatags:1]:
```suggestion
description: This example demonstrates creating a Kafka service to receive messages in an asynchronous manner.
```
Removed for consistency.

File: examples/kafka-producer-sasl/kafka_producer_sasl.metatags [hunk 6]
```diff
@@ -1,2 +1,2 @@
-description: BBE on how to configure a Kafka producer to use SASL/PLAIN authentication.
+description: This examples is on how to configure a Kafka producer to use SASL/PLAIN authentication.
```
Comment (praneesha) [inline examples/kafka-producer-sasl/kafka_producer_sasl.metatags:1]:
```suggestion
description: This example is on how to configure a Kafka producer to use SASL/PLAIN authentication.
```

File: examples/kafka-service-ssl/kafka_service_ssl.metatags [hunk 7]
```diff
@@ -1,2 +1,2 @@
-description: BBE on how to configure a Kafka listener to use SSL encryption.
+description: This example on how to configure a Kafka listener to use SSL encryption.
```
Comment (praneesha) [inline examples/kafka-service-ssl/kafka_service_ssl.metatags:1]:
```suggestion
description: This example is on how to configure a Kafka listener to use SSL encryption.
```

File: examples/sftp-client-send-file/sftp_client_send_file.metatags [hunk 8]
```diff
@@ -1,2 +1,2 @@
-description: This is a BBE on getting and putting file content on a remote SFTP server.
+description: This example demonstrates putting file content on a remote SFTP server.
```
Comment (praneesha) [inline examples/sftp-client-send-file/sftp_client_send_file.metatags:1]:
```suggestion
description: This example demonstrates adding file content to a remote SFTP server.
```
Check if the technical meaning got affected by the suggested change.

File: examples/sftp-service-receive-file/sftp_service_receive_file.metatags [hunk 9]
```diff
@@ -1,2 +1,2 @@
-description: This BBE is about receiving file/directory changes that occur in a remote SFTP server. It has a listener-based functionality for receiving these events.
+description: This example is about receiving file/directory changes that occur in a remote SFTP server. It has a listener-based functionality for receiving these events.
```
Comment (praneesha) [inline examples/sftp-service-receive-file/sftp_service_receive_file.metatags:1]:
```suggestion
description: This example demonstrates receiving file/directory changes that occur in a remote SFTP server. It has a listener-based functionality for receiving these events.
```
Changed for consistency.

File: examples/sftp-service-send-file/sftp_service_send_file.metatags [hunk 10]
```diff
@@ -1,2 +1,2 @@
-description: This BBE is about receiving file/directory changes that occur in a remote SFTP server and using the `ftp:Caller` to interact with the SFTP server.
+description: This example is about receiving file/directory changes that occur in a remote SFTP server and using the `ftp:Caller` to interact with the SFTP server.
```
Comment (praneesha) [inline examples/sftp-service-send-file/sftp_service_send_file.metatags:1]:
```suggestion
description: This example demonstrates receiving file/directory changes that occur in a remote SFTP server and using an ftp:Caller to interact with the SFTP server.
```

File: examples/ftp-service-receive-file/ftp_service_receive_file.metatags [hunk 11]
```diff
@@ -1,2 +1,2 @@
-description: This BBE is about receiving file/directory changes that occur in a remote FTP server. It has a listener-based functionality for receiving these events.
+description: This example is about receiving file/directory changes that occur in a remote FTP server. It has a listener-based functionality for receiving these events.
```
Comment (praneesha) [inline examples/ftp-service-receive-file/ftp_service_receive_file.metatags:1]:
```suggestion
description: This example demonstrates receiving file/directory changes that occur in a remote FTP server. It has a listener-based functionality for receiving these events.
```

File: examples/kafka-service-constraint-validation/kafka_service_constraint_validation.metatags [hunk 12]
```diff
@@ -1,2 +1,2 @@
-description: BBE on validating a payload according to the constraints defined in the payload record.
+description: This example is on validating a payload according to the constraints defined in the payload record.
```
Comment (praneesha) [inline examples/kafka-service-constraint-validation/kafka_service_constraint_validation.metatags:1]:
```suggestion
description: This example demonstrates validating a payload according to the constraints defined in the payload record.
```

File: examples/sftp-client-send-file/sftp_client_send_file.metatags (hunk 8 above)
Comment (dilanSachi) [reply examples/sftp-client-send-file/sftp_client_send_file.metatags:1]: The api is defined as `put`. So I think it's better if we say `putting`

**Review by praneesha — APPROVED** [review]

File: examples/ftp-service-send-file/ftp_service_send_file.metatags [hunk 13]
```diff
@@ -1,2 +1,2 @@
-description: This BBE is about receiving file/directory changes that occur in a remote FTP server and using the `ftp:Caller` to interact with the FTP server.
+description: This example demonstrates receiving file/directory changes that occur in a remote FTP server and using an ftp:Caller to interact with the FTP server.
```
Comment (TharmiganK) [inline examples/ftp-service-send-file/ftp_service_send_file.metatags:1]: Same confusion here, the description does not say about appending content to the added files. May be we can improve the two descriptions

File: examples/ftp-service-receive-file/ftp_service_receive_file.metatags [hunk 14]
```diff
@@ -1,2 +1,2 @@
-description: This BBE is about receiving file/directory changes that occur in a remote FTP server. It has a listener-based functionality for receiving these events.
+description: This example demonstrates receiving file/directory changes that occur in a remote FTP server. It has a listener-based functionality for receiving these events.
```
Comment (TharmiganK) [inline examples/ftp-service-receive-file/ftp_service_receive_file.metatags:1]: This example is on listening to file changes and reading the changes from the caller right? IMO the existing description is confusing

File: examples/sftp-service-receive-file/sftp_service_receive_file.metatags [hunk 15]
```diff
@@ -1,2 +1,2 @@
-description: This BBE is about receiving file/directory changes that occur in a remote SFTP server. It has a listener-based functionality for receiving these events.
+description: This example demonstrates receiving file/directory changes that occur in a remote SFTP server. It has a listener-based functionality for receiving these events.
```
Comment (TharmiganK) [inline examples/sftp-service-receive-file/sftp_service_receive_file.metatags:1]: Here also the description can be improved

File: examples/sftp-service-send-file/sftp_service_send_file.metatags [hunk 16]
```diff
@@ -1,2 +1,2 @@
-description: This BBE is about receiving file/directory changes that occur in a remote SFTP server and using the `ftp:Caller` to interact with the SFTP server.
+description: This example demonstrates receiving file/directory changes that occur in a remote SFTP server and using an ftp:Caller to interact with the SFTP server.
```
Comment (TharmiganK) [inline examples/sftp-service-send-file/sftp_service_send_file.metatags:1]: Here also the description can be improved

Comment (dilanSachi) [reply examples/sftp-service-send-file/sftp_service_send_file.metatags]: Fixed

File: examples/sftp-service-receive-file/sftp_service_receive_file.metatags (hunk 15 above)
Comment (dilanSachi) [reply examples/sftp-service-receive-file/sftp_service_receive_file.metatags]: Fixed

File: examples/ftp-service-receive-file/ftp_service_receive_file.metatags (hunk 14 above)
Comment (dilanSachi) [reply examples/ftp-service-receive-file/ftp_service_receive_file.metatags]: Fixed

File: examples/ftp-service-send-file/ftp_service_send_file.metatags (hunk 13 above)
Comment (dilanSachi) [reply examples/ftp-service-send-file/ftp_service_send_file.metatags]: Fixed

File: examples/kafka-consumer-ssl/kafka_consumer_ssl.metatags [hunk 17]
```diff
@@ -1,2 +1,2 @@
-description: BBE on how to configure a Kafka consumer to use SASL/PLAIN authentication.
+description: This example is on how to configure a Kafka consumer to use SASL/PLAIN authentication.
```
[the remark this answers is not in the document]
Comment (dilanSachi) [reply examples/kafka-consumer-ssl/kafka_consumer_ssl.metatags]: Fixed

**Review by praneesha — APPROVED** [review]

**Review by TharmiganK — APPROVED** [review]
LGTM

**Pull request merged**